doc/TOPP_tutorial/TOPP_scripting.doxygen

```diff
@@ -34,6 +34,269 @@
+                   and the ratio of #pepID/#ms2.
+              </TD>
+        </TR>
+
+        <TR>
+            <TH colspan="2">
+                <br><b>RTAlignment</b>
+            </TH>
+        </TR>
+        <TR>
+              <TD>
+                   <b>Essential input data:</b> PostFDR FeatureXML, trafoXML file
+              </TD>
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Brief: <br>
+                   This metric calculates the post map alignment retention time
+                   and sets new metavalues.<br>
+                   @b Description: <br>
+                   The RTAlignment metric checks what the retention time was before the alignment
+                   and how it is after the alignment. These two values are added to the metavalues
+                   in the peptideIdentification.
+              </TD>
+
+              <TD>
+                   <b> Changes in files:</b> <br>
+                   This metric adds the new metavalues 'rt_align' and 'rt_raw' to the peptideidentifications.<br>
+                   <b> Other Output:</b> <br>
+                   No additional output.
+              </TD>
+        </TR>
+        <TR>
+            <TH colspan="2">
+                <br><b>TIC</b>
+            </TH>
+        </TR>
+        <TR>
+              <TD>
+                   <b>Essential input data:</b> raw mzML file
+              </TD>
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Brief: <br>
+                   This metric calculates the total ion count of an MSExperiment.<br>
+                   @b Description: <br>
+                   Compute Total Ion Count and applies the resampling algorithm, if a bin size in RT seconds greater than 0 is given.
+                   All MS1 TICs within a bin are summed up.
+              </TD>
+
+              <TD>
+                   <b> Changes in files:</b> <br>
+                   This metric does not change anything in the data.<br>
+                   <b> Other Output:</b> <br>
+                   The TIC metric stores the TIC chromatograms.
+              </TD>
+        </TR>
+        <TR>
+            <TH colspan="2">
+                <br><b>TopNoverRT</b>
+            </TH>
+        </TR>
+        <TR>
+              <TD>
+                   <b>Essential input data:</b> PostFDR FeatureXML, raw mzML file
+              </TD>
+              <TD>
+                   <b>Output</b>
+              </TD>
+        </TR>
+        <TR>
+              <TD>
+                   @b Brief: <br>
+                   This metric Calculates the ScanEventNumber, finds all unidentified MS2-Spectra and adds them to the
+                   unassigned peptideIdentifications. Additionally it writes new metavalues in the peptideIdentification.<br>
```

Comment:
`adds corresponding empty peptide identifications as placeholders to the unassigned PepID list`